README.md

```diff
@@ -130,81 +161,101 @@ jobs:
 | `coverity_stream_name`        | Stream name in Coverity   | Mandatory     |
 | `coverity_install_directory`        | Directory path to install Coverity | Optional    |
-| `coverity_policy_view`        | ID number of a saved view to apply as a "break the build" policy. If any defects are found within this view when applied to the project, the build will be failed with an exit code. <br> Example: coverity_policy_view: 100001 </br>       | Optional    |
-| `coverity_automation_prcomment`        | To enable feedback from Coverity security testing as pull request comment. <br> Supported values: true or false </br> | Optional     |
-| `github_token` | It is mandatory to pass github_token parameter with required permissions. The token can be github specified secrets.GITHUB_TOKEN with required permissions. <br> Example:  github_token: ${{ secrets.GITHUB_TOKEN }}   </br>      | Mandatory if  coverity_automation_prcomment is set true. |
-
+| `coverity_policy_view`        | ID number/Name of a saved view to apply as a "break the build" policy. If any defects are found within this view when applied to the project, the build will be failed with an exit code. <br> Example: coverity_policy_view: 100001 </br>       | Optional    |
+| `coverity_automation_prcomment`        | To enable feedback from Coverity security testing as pull request comment. Merge Request must be created first from feature branch to main branch to run Coverity PR Comment. <br> Supported values: true or false </br> | Optional     |
+| `github_token` | GitHub Access Token <br> **Example:** github_token: ${{ secrets.GITHUB_TOKEN }} | Mandatory if coverity_automation_prcomment is set as true |
           
 ## Synopsys GitHub Action - Black Duck
-The Synopsys Action supports both self-hosted (e.g. on-prem) and Synopsys-hosted Black Duck Hub instances.
 
-No preparation is typically needed before running the pipeline. In the default Black Duck Hub permission model,
-projects and project versions are created on the fly and as needed.
+Synopsys Action supports both self-hosted (e.g. on-prem) and Synopsys-hosted Black Duck Hub instances.
 
-On pushes, a full "intelligent" Black Duck scan will be run. On pull requests, a "rapid" ephemeral scan will be run.
-A future release of the action will provide code review feedback for newly introduced findings to the pull request.
+In the default Black Duck Hub permission model, projects and project versions are created on the fly as needed.
 
-We recommend configuring sensitive data like access tokens and even URLs, using GitHub secrets.
+The action will download the Bridge and Detect CLIs, run a SCA scan, and optionally break the build.
 
+On pushes, a full **Intelligent** Black Duck scan will be run. On pull requests, a **Rapid** ephemeral scan will be run.
+A future release of the action will provide code review feedback for newly introduced findings to the pull request.
```

Comment:
for BD and coverity we already have pr comment so  A future release of the action will provide code review feedback for newly introduced findings to the pull request.  we can remove. same for coverity

Reply:
updated